scripts/time_plot.py

```diff
@@ -1497,23 +1498,23 @@ def handle_websock_event(handlerkey,*args):
                 for thishandler in websockrequest_username.keys():
                     if (websockrequest_username[thishandler]==username):
                         send_websock_cmd('ApplyViewLayout('+'["'+'","'.join([fig['figtype'] for fig in html_viewsettings[username]])+'"]'+','+str(html_layoutsettings[username]['ncols'])+')',thishandler)
-        elif (args[0].startswith('holo')):#accepts 'holo', 'holo 3' where 3 is m003 as explicit reference antenna, else first track antenna is chosen as reference antenna
+        elif (args[0].startswith('holo')):#accepts 'holohh', 'holovv', 'holohv', 'holovh', 'holo 3' where 3 is m003 as explicit reference antenna, else first track antenna is chosen as reference antenna
             logger.info(repr(args))
             antnumbers=[int(antnumberstr[1:]) for antnumberstr in telstate_antenna_mask]#determine all available inputs
             if (len(antnumbers)==0):
                 send_websock_cmd('logconsole("No antenna inputs found or specified",true,true,true)',handlerkey)
             else:
-                cbid=str(telstate['sdp_capture_block_id'])
+                cbid=telstate['sdp_capture_block_id']#this is a string
                 obs_params_key=telstate.join(cbid, 'obs_params')
                 obs_params=telstate.get(obs_params_key, {})
                 scan_ants=[]
                 track_ants=[]
-                if 'scan_ants' in obs_params:
-                    scan_ants=obs_params['scan_ants'].split(',')
-                if 'track_ants' in obs_params:
-                    track_ants=obs_params['track_ants'].split(',')
-                if 'scan_ants_always' in obs_params:
-                    scan_ants_always=obs_params['scan_ants_always'].split(',')
+                if b'scan_ants' in obs_params:
+                    scan_ants=to_str(obs_params[b'scan_ants']).split(',')
+                if b'track_ants' in obs_params:
+                    track_ants=to_str(obs_params[b'track_ants']).split(',')
+                if b'scan_ants_always' in obs_params:
+                    scan_ants_always=to_str(obs_params[b'scan_ants_always']).split(',')
```

Comment:
Eventually we'll get CAM to start using Python 3 and then obs_params will use strings. Instead of changing all this code to use bytes, you can just change how you get obs_params to `obs_params = to_str(telstate.get(obs_params_keys, {}))`, which will convert all the keys and values to strings.

Reply:
yes, they should all be changed in a future PR, to to_str

Reply:
Since this is the first time you're using to_str, you cshould start using it right in this PR i.e. use `to_str` when you get obs_params out of telstate instead of on each access to it.

Reply:
To be clear, I'm not suggesting you should try to roll out to_str to the rest of the code - just update the code that's new in this PR.